src/cfn.py

```diff
@@ -75,7 +75,14 @@ def update_infrastructure(stackname):
     but without any software being on it)
 
     Moreover, EC2 instances must be running while this is executed or their
-    resources like PublicIP will be inaccessible"""
+    resources like PublicIP will be inaccessible.
+
+    Allows to skip EC2, SQS, S3 updates by passing `skip=ec2\\,sqs\\,s3`"""
```

Comment:
this weird notation is needed because `,` separates different arguments to `update_infrastructure` (which has 2: `stackname` and `skip`). Can replace with another separator if we prefer one

Reply:
can we do `./bldr update-infrastructure:project--instance,skip="ec2,sqs,s3"` ?

Reply:
approved, but lets fix up that escaped notation if possible

Reply:
this works

```python
@task
def foo(args):
    print(args.split(" "))
```
and 
```bash
$ ./bldr foo:args="1 2 3"
['1', '2', '3']

done
```